src/yaml/mapping.cr

```diff
@@ -216,7 +216,7 @@ module YAML
 
   # This is a convenience method to allow invoking `YAML.mapping`
   # with named arguments instead of with a hash/named-tuple literal.
-  macro mapping(**properties)
-    ::YAML.mapping({{properties}})
+  macro mapping(**_properties_)
```

Comment:
I'm pretty sure that changing these 2 lines is all that's needed. No idea why the other PR went that far.

Reply:
I don't think you need to change this macro, the splat arg name cannot be used in a call IIRC, so cannot clash

Reply:
That too, but I guess it means this is just a bug in macro splat

Reply:
Nevermind, forget what I said, I didn't know one could do https://carc.in/#/r/37a9 to mention that splat

Reply:
@oprypin Both need to be renamed. Otherwise `mapping(properties: String)` would still match the other signature.